src/components/metadata/metadata.test.js

```diff
@@ -66,6 +67,40 @@ describe('MetaData', () => {
       // Should show all 20 values
       expect(parametersRow().find('.pipeline-metadata__value').length).toBe(20);
     });
+
+    it('when pretty name is turned off the metadata title displays the full node name', () => {
+      const props = {
+        nodeId: salmonTaskNodeId,
```

Comment:
It would be better to set up a node that has a more distinctive difference between their pretty name and non-pretty name other than `salmon` - the `params:pipeline100.data_science.plankton` node would work much better.